packages/core/mesh/network-manager/src/transport/memory-transport.test.ts

```diff
@@ -72,3 +78,15 @@ describe('MemoryTransport', () => {
     );
   });
 });
+
+const createTransportOptions = (options: Partial<TransportOptions>): TransportOptions => {
+  return {
+    initiator: false,
```

Comment:
is the flag ignored now?

Reply:
yeah, considered removing it, but it's used in other implementations